go/internal/feast/local.go

```diff
@@ -44,8 +46,7 @@ func (r *LocalRegistryStore) UpdateRegistryProto(rp *core.Registry) error {
 }
 
 func (r *LocalRegistryStore) Teardown() error {
-	err := os.Remove(r.filePath)
-	if err != nil {
+	if err := os.Remove(r.filePath); err != nil {
 		return err
 	}
 	return nil
```

Comment:
Can't this just be changed to `return os.Remove(r.filePath)`?

Reply:
good catch